install.zsh

```diff
@@ -74,9 +74,9 @@ if [[ -z $ZSH_CUSTOM ]]; then
 fi
 
 # Linking
-log "Linking $SPACESHIP to $DIST..."
+log "Copying $SPACESHIP to $DIST..."
 mkdir -p "$(dirname $DIST)"
-ln -sf "$SPACESHIP" "$DIST"
+cp -f "$SPACESHIP" "$DIST"
```

Comment:
It's okay for `/tmp`, however it would be not OK for case when we install it through NPM.